docs/self-managed/setup/deploy/amazon/openshift/terraform-setup.md

```diff
@@ -296,96 +311,241 @@ To set up a ROSA cluster, certain prerequisites must be configured on your AWS a
+
+1. For each client connecting to the cluster, assign a unique name in `client_key_names` to simplify certificate revocation.
+
+1. By default, VPN access is allowed from any IP address. You may restrict access by adjusting the `vpn_allowed_cidr_blocks` variable.
+
+1. Network designs vary; please review and adjust the configuration to fit your topology.
+
+1. Customize the VPN module by referring to the [VPN module documentation](https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-private-link/aws/modules/vpn/README.md).
+
+#### Outputs
+
+The module stores certificates and VPN client configurations in the Terraform state. The next section explains how to retrieve and use client configurations.
+
+#### Execution
+
+1. Generate a Terraform plan for the VPN Client Endpoint configuration.
+   This will use the private subnets of the designated VPC:
+
+   ```bash
+   # describe what will be created
+   terraform plan -out vpn.plan \
+    -var vpc_id="$CLUSTER_VPC_ID"
+   ```
+
+1. Review and apply the plan to create the resources:
+
+   ```bash
+   terraform apply vpn.plan     # creates the resources
+   ```
+
+Creation of the VPN Client Endpoint typically takes about 10 minutes. After completion, the client configurations will be available in the Terraform output `vpn_client_configs`.
+
 ### Reference files
 
 Depending on the installation path you have chosen, you can find the reference files used on this page:
 
-- **Standard installation:** [Reference Files](https://github.com/camunda/camunda-deployment-references/tree/main/aws/openshift/rosa-hcp-single-region/)
+- **Standard installation:** [Reference Files](https://github.com/camunda/camunda-deployment-references/tree/feature/openshift-private-link/aws/openshift/rosa-hcp-single-region/)
 
 ## 2. Preparation for Camunda 8 installation
 
+### Access to the private network using the VPN
+
+This section applies if you have previously created a private cluster and want to access it using the [VPN module configured earlier](#vpn-module-setup).
+
+1. Navigate to the VPN module directory (`vpn`):
+
+    ```bash
+    pwd
+
+    # Example output:
+    # ./camunda-deployment-references/aws/openshift/rosa-hcp-single-region/terraform/vpn/
+    ```
+
+1. Generate your client’s VPN configuration file. This file is compatible with [OpenVPN (ovpn)](https://openvpn.net/) format:
+
+    ```bash reference
+    https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-private-link/aws/openshift/rosa-hcp-single-region/procedure/gather-vpn-config.sh
+    ```
+
+1. Import the generated configuration file (`my-client.ovpn`) into your preferred OpenVPN client:
+
+   - [Official AWS VPN Client](https://docs.aws.amazon.com/vpn/latest/clientvpn-user/connect-aws-client-vpn-connect.html)
+   - [Other OpenVPN Clients](https://docs.aws.amazon.com/vpn/latest/clientvpn-user/connect.html)
```

Comment:
for me it didn't work with default openvpn.net client.
While it could connect, I didn't have access to the internet nor the cluster.

I switched to AWS VPN Client, same config, worked immediately.

So I'm not sure whether any other OpenVPN client will work. At least the "official" OpenVPN Client did not work for me.

Reply:
maybe instead of `preffered` to `comaptible`?

Tunnelblick is listed as part of the AWS list, so I guess it's supported.
While others I guess seem to not support whatever AWS is doing differently 🤷.